### PR TITLE
ET50-Version 2

### PR DESCRIPTION
**Pull request by DispySK**
Updated the code, created a new method that tracks what key is pressed and another method that handles the movement. 

Now there is no delay between clicking a button and moving :))))))))

### CODE REVIEW

File: src/main/main.java [hunk 1]
```diff
@@ -23,27 +24,41 @@ public void init() {
     public void loop() { //Naufal the goat
```
Comment (NaufalS9) [inline src/main/main.java]: Would be nice if this comment deleted 😁

Comment (NaufalS9) [reply src/main/main.java]: Just merge it to main after, i would say

**Review by NaufalS9 — CHANGES_REQUESTED** [review]

File: src/main/main.java (hunk 1 above)
Comment (patricksanto) [reply src/main/main.java]: @DispySK please guy!

Comment (DispySK) [reply src/main/main.java]: removed

**Pull request merged**